recipe/meta.yaml

```diff
@@ -1,16 +1,16 @@
-{% set version = "1.0.2" %}
+{% set version = "1.0.3" %}
 
 package:
   name: qtawesome
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/q/qtawesome/QtAwesome-{{ version }}.tar.gz
-  sha256: 771dd95ac4f50d647d18b4e892fd310a580b56d258476554c7b3498593dfd887
+  sha256: d37bbeb69ddc591e5ff036b741bda8d1d92133811f1f5a7150021506f70b8e6e
 
 build:
   number: 0
-  skip: true  # [ppc64le]
+  skip: true  # [ppc64le or s390x]
   noarch: python
   script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
 
```

Comment:
There are missing packages in the host section: `setuptools`, `wheel`

Reply:
`Setuptools` and `wheels` were added to the recipe

https://github.com/AnacondaRecipes/qtawesome-feedstock/blob/update_to_1.0.3/recipe/meta.yaml#L21&L22